### PR TITLE
Markgermline rank

### PR DESCRIPTION
**Pull request by ViktorHy**
Added support for more in-depth assigning of GERMLINE to paired samples. This should make it able to have a lot of genes available for germline analysis but still not get overwhelmed by variants. Right now it disqualifies variants with high gnomad frequencies. It tries to include variants with severe conseqences on protein and variants with clinvar annotations.

All three categories are modifiable in a JSON-config as well as defining all GERMLINE genes of interest.

### CODE REVIEW

**Comment by ViktorHy** [comment]
Have not tested tumor-only cases. TODO

File: bin/mark_germlines.pl [hunk 1]
```diff
@@ -98,11 +104,103 @@ sub print_header {
...
+sub mini_rank {
+    my $var = shift;
+    my $rank = shift;
+    my %rank = %$rank;
+    my $score = 0;
+
+    ## check clinvar ##
+    # if only one annotation handle differently, this is to support sole Benign annotations
+    # otherwise find highest score acoring to rank->clinvar from JSON. 
+    # This let's the user define some value and let other be 0. This is ugly?
```
Comment (sailepradh) [inline bin/mark_germlines.pl:131]: took some time to understood but makes sense

**Review by sailepradh — COMMENTED** [review]
Reviwed the code and logic for the scoring of the variants-- Might need some more information in the mini_rank function and scores for different consequence_score

**Comment by ViktorHy** [comment]
I have now tested this for paired solid samples and for one paired parp sample and it generates correct files. Sample tested has been noted in internal document on teams

**Review by ramsainanduri — APPROVED** [review]
LGTM

**Pull request merged**